### PR TITLE
email configuration and transfer amount

### PR DESCRIPTION
**Pull request by ganeshzilpe**

### CODE REVIEW

File: src/main/java/com/asu/cse545/group12/domain/Users.java [hunk 1]
```diff
@@ -58,12 +58,12 @@ public String toString() {
 	@Column(name = "GENDER")
 	private String gender;
 	
-	@NotNull
+	//@NotNull
     //@NotEmpty//(message = "Please enter your Username.")
```
Comment (chasethenag420) [inline src/main/java/com/asu/cse545/group12/domain/Users.java:62]: Shouldn't we uncomment @Notempty ?

File: src/main/java/com/asu/cse545/group12/domain/Users.java [hunk 2]
```diff
@@ -58,12 +58,12 @@ public String toString() {
...
-	@NotNull
+	//@NotNull
     //@NotEmpty//(message = "Please enter your Username.")
 	@Column(name = "USERNAME")
 	private String userName;
 	
-	@NotNull
+	//@NotNull
     //@NotEmpty//(message = "Please enter your password.")
    // @Size(min = 6, max = 15, message = "Your password must between 6 and 15 characters")
```
Comment (chasethenag420) [inline src/main/java/com/asu/cse545/group12/domain/Users.java:68]: why these validations are commented? im not sure who commented though.

File: src/main/webapp/WEB-INF/jsp/admin.jsp [hunk 3]
```diff
@@ -1,77 +1,105 @@
...
+			<div class="col-md-3 column margintop20">
+				<ul class="nav nav-pills nav-stacked">
+					<li id="home" class="active"><a 
+						, href="#"><span class="glyphicon glyphicon-chevron-right"></span>
+							Home</a></li>
+					<li id="viewProfile"><a href="#"><span
+							class="glyphicon glyphicon-chevron-right"></span> View Profile</a></li>
+					<li id="createEmployee"><a href="#"><span
+							class="glyphicon glyphicon-chevron-right"></span> Create Employee</a></li>
+					<li id="modifyEmployee"><a  href="#"><span
```
Comment (chasethenag420) [inline src/main/webapp/WEB-INF/jsp/admin.jsp:44]: We should move this to search user and then modify or delete based on approvals

File: src/main/java/com/asu/cse545/group12/domain/Users.java (hunk 1 above)
Comment (ganeshzilpe) [inline src/main/java/com/asu/cse545/group12/domain/Users.java:62]: We are going to handle it in validator. Currently we are using same form
for creating external user and internal user and system administrator
doesn't input username and password while creating internal user. So I
comment it

**Pull request merged**